app.js

```diff
@@ -1,26 +1,66 @@
-var createError = require('http-errors');
-var express = require('express');
-var path = require('path');
-var cookieParser = require('cookie-parser');
-var logger = require('morgan');
+const createError = require("http-errors");
+const express = require("express");
+const path = require("path");
+const cookieParser = require("cookie-parser");
+const logger = require("morgan");
+const mongoose = require("moongoose");
+const databaseConnection = require("./config/dbConfig");
+const bodyParser = require("body-parser");
```

Comment:
you left this out `app.use(bodyParser.urlencoded({
    extended: true
}))`
Hence you will not be able to get the parsed body from the post endpoint. You will be able to access get endpoint but your req.body in post endpoints will be undefined